service/orchestrator/toe.go

```diff
@@ -206,8 +275,10 @@ func (svc *Service) ListControlsInScope(ctx context.Context, req *orchestrator.L
 		return nil, service.ErrPermissionDenied
 	}
 
+	conds = append(conds, "target_of_evaluation_cloud_service_id = ? AND target_of_evaluation_catalog_id = ?", req.CloudServiceId, req.CatalogId)
```

Comment:
Need to add authz here as well. Something like:
```
// Check, if this request has access to the cloud service according to our authorization strategy.
	if !s.authz.CheckAccess(ctx, service.AccessRead, req) {
		return nil, service.ErrPermissionDenied
	}
```

and then make sure that ListControlsInScopeRequest implements `GetCloudServiceId` in api/orchestrator/orchestrator.go

Reply:
This method already has the check. See line 274

Reply:
Do we want to implement all auth checks here or in another PR?

Reply:
> Do we want to implement all auth checks here or in another PR?

Might as well fix it here for those methods we touch. I think may comment might have been to another function. I am pretty certain that one of them was still missing the check.

Reply:
Ok, I will check that.

Reply:
I've added the auth check to `ListTargetsOfEvaluation` and `CreateTargetOfEvaluationRequest`.

Reply:
I took the liberty in adding the remaining authz checks for all functions in the `toe.go` while we are changing this file anyway. Looks good from my side. one last check from your side @anatheka ?

Reply:
Perfect, from my side it also fits.

Reply:
Maybe you can also have a look at the UI PR clouditor/clouditor-ui#317